hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/AbstractAbfsIntegrationTest.java

```diff
@@ -588,4 +591,38 @@ protected boolean isAppendBlobEnabled() {
   protected AbfsServiceType getAbfsServiceType() {
     return abfsConfig.getFsConfiguredServiceType();
   }
+
+  /**
+   * Create directory with implicit parent directory.
+   * @param path path to create. Can be relative or absolute.
+   */
+  protected void createAzCopyDirectory(Path path) throws Exception {
+    String sasToken = getRawConfiguration().get(FS_AZURE_TEST_FIXED_SAS_TOKEN);
+    AzcopyToolHelper azcopyHelper = new AzcopyToolHelper(
+        getAccountName(), getFileSystemName(), sasToken);
+    azcopyHelper.createFolderUsingAzcopy(getPathFromRoot(path));
+    Assertions.assertThat(isImplicitDirectory(path, getFileSystem(),
```

Comment:
why we need to assert this. This is an az-copy functionality. We can just move ahead with assurance that it would be creating implicit dirs.

My main concern is that we are adding this assertion at every create and mkdir for an implicit path. This might make tests  slow.

Reply:
Makes sense.
Wanted to make sure azcopy tool is working as expected.
Added separate test to assert that